### ISSUE TITLE
Application Form: Nickname

### ISSUE THREAD

**Issue by JamesTheClarke** [CntoDev/level-up#128]
The nickname field in the application could be improved with the following changes:

- [ ] The explanation text should change from `Use the same nickname as in Enjin` to `This will be your uniform nickname for all CNTO platforms. Limited to max. 2 words in length, each capitalised, max. 10 characters in total (spaces included), avoid odd formatting.`
- [ ] The field should automatically check for the correct guidelines and give a red error message if the chosen nickname does not comply with our naming convention.

Detailed nicknaming convention:

- length of max 10 characters (spaces included)
- can include max 1 space
- can not include any numbers
- can not include any symbols / special characters
- can not be formatted with odd / hard to read capitalisation (e.g. CoWbOyKiLlAh)
- can not be all caps if longer than three characters (e.g. CLARKE) 
- 1st letter of nickname needs to be capitalised, if the nickname is 2 words then both need to be capitalised

**Comment by enrico-ghidoni** [comment]
The help text was changed to `Use the same nickname as in Enjin` for UI reasons. Here's how it would look with the full text and in my opinion it's just ugly. So since the naming conventions are already enforced on Enjin it would be useless to point them out again and I figured telling applicants to use what is an already validated nickname has the same effect without compromising the whole form look.

![image](https://user-images.githubusercontent.com/8669492/153175808-73835f20-7f99-45c5-96db-53e37295ca9a.png)

Concerning nickname validation, it already happens and all the checks are in place. It's not asynchronous (as in, the nickname is verified once you submit the form and not when you type in the nickname itself).
The only missing check is the odd capitalization as it's very subjective. Instead, we can force it to be always in lower case except for the first letter which must be capitalized (which is already enforced).
